eng/updateversions.proj

```diff
@@ -6,6 +6,10 @@
 
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 
+  <Import Project="$(NuGetPackageRoot)\microsoft.dotnet.versiontools.tasks\$(MicrosoftDotNetVersionToolsTasksPackageVersion)\build\Microsoft.DotNet.VersionTools.Tasks.props" />
```

Comment:
Do we still need to add `Microsoft.DotNet.VersionTools.Tasks` as a repo dependency for auto-updates? (I don't see it in `Version.Details.xml`)